en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,274 @@
+It will then search for tasks options in ``./grunt_tasks/options/`` and ``./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks/Options/``.
+As you run grunt without any parameter it will then launch the Default task.
+
+.. _add an external bundle using grunt:
+
+Adding an external bundle using grunt
+-------------------------------------
+
+If you take a closer look at the ``open-orchestra-cms-bundle``, you will see it does not contain
+the media administration part. To manage the media, you have to add the ``open-orchestra-media-admin-bundle``.
+
+This bundle comes with its own javascript and css files that need to be added to the website assets
+files. Assuming you have installed the bundle using composer, you still have to configure grunt to
+use the bundle assets.
+
+For that purpose, you need to update 3 parts of the grunt configuration:
+
+- load the new tasks
+- load the new task options
+- tell somewhere when these tasks must be run
+
+To load the task, you have to use ``grunt.loadTasks`` on the correct directory. In our case you have
+to update the ``Gruntfile.js`` file with this:
+
+.. code-block:: javascript
+
+    // line 5
+    grunt.loadTasks('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks');
+
+To load task options, you have two possibilities, depending on what you want to do:
+
+- to load a single file, use the ``loadFileConfig`` function
+- to load all files in a directory, use the ``loadDirConfig`` function
+
+In both cases, you have to merge the resulting object to the main configuration object.
+
+For the ``open-orchestra-media-admin-bundle``, we want to load all files from a single directory,
+so it looks like:
+
+.. code-block:: javascript
+
+    // line 13
+    config = merge.recursive(true, config, loadDirConfig('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks/Options/'));
+
+``Grunt`` will now be aware of the different tasks present in the ``MediaAdminBundle``.
+
+Some tasks, such as the symlinks generation, will be automatically launched. Other tasks will
+require some configuration modifications, such as js and css concatenation, due to their inner nature.
+All the Back Office javascript files are append in one big file, and the same operation is done for
+stylesheets. This is done in two passes (two similar passes for both js and css).
+
+The first pass groups files by functionnalities. For instance all the javascript files related to theme
+are put in the same javascript file.
+The second pass groups all these 'functionnality' files in a unique file. The Back Office uses that
+final javascript (or stylesheet) file.
+
+In the case of our ``open-orchestra-media-admin-bundle``, the file concat.mediajs.js describes the
+first pass of the js concatenation: it glues all media javascripts together. The file concat.mediacss.js
```

Comment:
first part of the javascript concatenation. (the rest of the sentence is too much)